utils/Redirect-map.json

```diff
@@ -207,7 +207,7 @@
   "/basic-docs/atomicdex/atomicdex-tutorials/api-walkthrough.html": "/komodo-defi-framework/tutorials/api-walkthrough/",
   "/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.html": "/komodo-defi-framework/tutorials/coins-file-update/",
   "/basic-docs/atomicdex/atomicdex-tutorials/how-to-become-a-liquidity-provider.html": "/komodo-defi-framework/tutorials/how-to-become-a-liquidity-provider/",
-  "/basic-docs/atomicdex/atomicdex-tutorials/how-to-compile-mm2-from-source.html": "/komodo-defi-framework/tutorials/how-to-compile-mm2-from-source/",
+  "/basic-docs/atomicdex/atomicdex-tutorials/how-to-compile-kdf-from-source.html": "/komodo-defi-framework/tutorials/how-to-compile-kdf-from-source/",
```

Comment:
revert "/basic-docs/atomicdex/atomicdex-tutorials/how-to-compile-kdf-from-source.html" to "/basic-docs/atomicdex/atomicdex-tutorials/how-to-compile-mm2-from-source.html"

Reply:
fixed in https://github.com/KomodoPlatform/komodo-docs-mdx/pull/364/commits/030d59e9e3f69d75b05ff07115263aacb8e4388a